### ISSUE TITLE
Standardize proposal numbers 1-9

### ISSUE THREAD

**Issue by willhennessy** [Uniswap/interface#2866]

**Comment by moodysalem** [comment]
i don't think this is a good idea, it makes it even more confusing that `/<bravo address>/1` would point to proposal 9, and also breaks old urls pointing to `/<bravo address>/9`

**Comment by willhennessy** [comment]
Should we do away with the /<bravo address>/  URL pattern then? ideally the
URL would simply be /proposal/1

I think the short-term transition cost is worth paying up front at this
early stage, so that future users be a more clear experience

Problem statement:  people are starting to talk about Uniswap proposal
numbers like UNI9 (getty tweet
<https://twitter.com/getty_hill/status/1458593882921443335>). Other
interfaces like Tally use this notation too. It will be more clear if we
align and abstract the contract details away from the user
